application/github/v0/config/tasks.json

```diff
@@ -0,0 +1,1607 @@
+        "commit"
+      ],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_GET_ALL_ISSUES": {
+    "description": "Get the list of all issues in a repository,This can be a pull request or a general issue, and you can tell by the `is_pull_request` field.",
+    "instillShortDescription": "Get the list of all issues in a repository",
+    "input": {
+      "description": "Please input the repository name and owner",
+      "instillUIOrder": 0,
+      "instillEditOnNodeFields": [
+        "owner",
+        "repository"
+      ],
+      "properties": {
+        "$ref": "#/$defs/repository_info",
+        "state": {
+          "default": "open",
+          "title": "State",
+          "description": "State of the issues, can be one of: open, closed, all. Default is open",
+          "enum": [
+            "open",
+            "closed",
+            "all"
+          ],
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 10,
+          "type": "string"
+        },
+        "sort": {
+          "default": "created",
+          "title": "Sort",
+          "description": "Sort the issues by created, updated, popularity, or long-running. Default is created",
+          "enum": [
+            "created",
+            "updated",
+            "popularity",
+            "long-running"
+          ],
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 11,
+          "type": "string"
+        },
+        "direction": {
+          "default": "desc",
+          "title": "Direction",
+          "description": "Direction of the sort, can be one of: asc, desc. Default is asc",
+          "enum": [
+            "asc",
+            "desc"
+          ],
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 12,
+          "type": "string"
+        },
+        "since": {
+          "default": "2021-01-01T00:00:00Z",
+          "title": "Since",
+          "description": "Only issues updated at or after this time are returned. Default is 2021-01-01T00:00:00Z",
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference"
+          ],
+          "instillUIOrder": 13,
+          "type": "string"
+        },
+        "no_pull_request": {
+          "title": "No Pull Request",
+          "description": "Whether to not include pull requests in the issues. Default is false",
```

Comment:
Question
What if it is true?
I am not sure if the output in json deals with this case

Will come back to check it after reading more code again as well.

Reply:
If it is true, it will filter out all items that are actually a pr. Since issue and pr use the same indexing system in github, the API returns all relevant objects (issues and pr). I leave this param for user to simplify the response.